### DISCUSSION TITLE
memory/rumble pack

### DISCUSSION THREAD

**Discussion by thechad666** [simple64/simple64#343] [Ideas]
great work!! i was wondering how hard it would be to add support for save/rumble pack swapping or to allow  save/load state to be assigned to unused controller buttons. would be really cool if when you chose save/load in game it automatically created save/load but pretty sure that's not possible.

**Comment by Jj0YzL5nvJ** [comment]
[Mempak switch](https://imgur.com/a/KVBg3RH) in upstream... I don't know if `simple64-input-qt` has it implemented...

**Comment by wiesshund** [comment]
> great work!! i was wondering how hard it would be to add support for save/rumble pack swapping or to allow save/load state to be assigned to unused controller buttons. would be really cool if when you chose save/load in game it automatically created save/load but pretty sure that's not possible.

i would like to see this as well
is a little hard to go out of full screen, to ui menu to controller settings, then pick the right pak, then save game, then go back to rumble, then back to full screen, with a gamepad.
